Fix `test_multipart_copy_versioned`

This test is failed on bucket versioning status asset. 
```
AssertionError: 'Suspended' != None
```

```py
def test_multipart_copy_versioned():
    src_bucket_name = get_new_bucket()
    dest_bucket_name = get_new_bucket()

    dest_key = "mymultipart"
    check_versioning(src_bucket_name, None)  # <- FAILED THERE
    ...
```
It expects no value when checking versioning status in the fresh new bucket, but got `Suspended`.

See this code: https://github.com/nspcc-dev/neofs-s3-gw/blob/3f0dbcc995344c4c6171cf4f22358aea61d1c071/api/handler/versioning.go#L77-L88
This function takes non nil value and returns `Suspended`.

--- a/api/data/info.go
+++ b/api/data/info.go
@@ -46,6 +46,7 @@ type (
 
 	// BucketSettings stores settings such as versioning.
 	BucketSettings struct {
+		IsNoneStatus      bool
 		VersioningEnabled bool                     `json:"versioning_enabled"`
 		LockConfiguration *ObjectLockConfiguration `json:"lock_configuration"`
 	}

--- a/api/handler/versioning.go
+++ b/api/handler/versioning.go
@@ -34,6 +34,7 @@ func (h *handler) PutBucketVersioningHandler(w http.ResponseWriter, r *http.Requ
 	}
 
 	settings.VersioningEnabled = configuration.Status == "Enabled"
+	settings.IsNoneStatus = false
 
 	p := &layer.PutSettingsParams{
 		BktInfo:  bktInfo,
@@ -76,7 +77,7 @@ func (h *handler) GetBucketVersioningHandler(w http.ResponseWriter, r *http.Requ
 
 func formVersioningConfiguration(settings *data.BucketSettings) *VersioningConfiguration {
 	res := &VersioningConfiguration{}
-	if settings == nil {
+	if settings.IsNoneStatus {
 		return res
 	}
 	if settings.VersioningEnabled {

--- a/api/layer/system_object.go
+++ b/api/layer/system_object.go
@@ -245,6 +245,7 @@ func (n *layer) GetBucketSettings(ctx context.Context, bktInfo *data.BucketInfo)
 		if !errors.IsS3Error(err, errors.ErrNoSuchKey) {
 			return nil, err
 		}
+		settings.IsNoneStatus = true
 	} else if err = json.Unmarshal(obj.Payload(), settings); err != nil {
 		return nil, err
 	}

